src/ui/molecules/networkGraph/ecs/schema.ts

```diff
@@ -0,0 +1,37 @@
+import { number, registerSchema, string } from '@javelin/ecs'
+
+export enum Schemas {
+  Node = 0,
+  Link = 1,
+  Position = 2,
+  Mesh = 3,
+  Label = 4
```

Comment:
In enums, you don't need to specify = 0, =1 etc... because they are auto-incrementing from 0.
It is however useful if you want the starting value to be other than 0.
So you can just write: 
```
export enum Schemas {
  Node,
  Link,
  Position,
  Mesh,
  Label
  }
```
it should work.

Reply:
That's right!